vendor/modules.txt

```diff
@@ -426,9 +426,38 @@ github.com/hashicorp/go-azure-sdk/resource-manager/logic/2019-05-01/workflowtrig
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/codeversion
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/componentcontainer
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/componentversion
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/datacontainer
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/datacontainerregistry
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/datastore
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/dataversion
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/dataversionregistry
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/environmentcontainer
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/environmentversion
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/job
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/machinelearningcomputes
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/modelcontainer
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/modelversion
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/onlinedeployment
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/onlineendpoint
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/operationalizationclusters
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/outboundnetworkdependenciesendpoints
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/privateendpointconnections
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/proxyoperations
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/quota
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/registrymanagement
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/schedule
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/v2workspaceconnectionresource
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/virtualmachinesizes
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/workspaceprivateendpointconnections
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/workspaceprivatelinkresources
+github.com/hashicorp/go-azure-sdk/resource-manager/machinelearningservices/2023-04-01/workspaces
```

Comment:
we're only importing 3 of the SDK's here, are we planning on using more of these? If not, it's probably not worth using the Meta Client for this one?

Reply:
thanks for you suggestion, I'm going to use one(or two) new sdk for the new resource. I have updated this PR to use separate clients insted.